cmd/gen-stdlib-allowlists/main_test.go

```
@@ -168,3 +276,2099 @@ func TestNormalizeMatchesRuleSemantics(t *testing.T) {
+	scanRoot := t.TempDir()
+	mustMkdirAll(t, filepath.Join(scanRoot, "src", "foo"))
+	mustWriteFile(t, filepath.Join(scanRoot, "src", "foo", "foo.go"),
+		"package foo\nfunc F() {}\n")
+	tests := []struct {
+		name    string
+		flags   *cliFlags
+		wantErr bool
+	}{
+		{
+			name:    "unknown kind rejected",
+			flags:   &cliFlags{goroot: runtime.GOROOT(), kind: "bogus"},
```

golangci-lint (2.12.2)

[error] SA1019: runtime.GOROOT has been deprecated since Go 1.24: The root used during the Go build will not be meaningful if the binary is copied to another machine. Use the system path to locate the “go” binary, and use “go env GOROOT” to find its GOROOT. 

(staticcheck)